roofit/roofitcore/src/TestStatistics/LikelihoodJob.h

```diff
@@ -71,6 +71,12 @@ class LikelihoodJob : public MultiProcess::Job, public LikelihoodWrapper {
 
    LikelihoodType likelihood_type_;
    std::size_t N_tasks_at_workers_ = 0;
+
+   // warning: don't use the following values directly, use the getters instead!
+   std::size_t N_event_tasks_;
+   std::size_t N_component_tasks_;
```

Comment:
```
   std::size_t n_event_tasks_;
   std::size_t n_component_tasks_;
```
Please follow the code convention of lower case data members. To be consistent with the other variables in this class, maybe use `snake_case_`. Also, the `N_tasks_at_workers_` should be updated.

Reply:
I can do this for these members, but note that the class itself is inconsitent anyway. For instance, there are both dromedary case and snake case member functions. Hope that's not a problem, because otherwise this PR will quickly blow out of proportions into a major refactoring effort ;)

Reply:
Yes I know, it's already inconsistent :) But at least there should be either a trailing or a leading underscore somewhere to indicate a private member, and lowercase in the beginning to indicate a variable. Snake case or camel case is secondary